### DISCUSSION TITLE
Find out which user account owns organization

### DISCUSSION THREAD

**Discussion by DTDID** [community/community#35609] [New to GitHub]
My regular user account (12@ab.cd) is not the owner of an organization (XYZ) I must have created in the past. I can see the old logo of this project, but this organization is not in the list of organizations my user account (12@ab.cd) is a part of/owns.

I must have created this organization (XYZ) with another user account, something like user@xyz.com. I’ve tried to reset password for a few possible email addresses, but no luck. These seem not to be a valid account.

How can I find out which user account "owns" this organization account?
Is there any way to send a message to this organization, so I can see which email address (user account) receives that message? 

I understand that the user account/email address is hidden, but a simple ping might tell me which email address I may have used to create this organization in the past.

Or asked in another way… how do I reach an unknown/hidden owner of an organization?

**Comment by fadyat** [comment]
If I correctly understand, you need [this](https://docs.github.com/en/enterprise-cloud@latest/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/viewing-peoples-roles-in-an-organization)

**Reply by DTDID** [reply]
Unfortunately this doesn’t seems to be the solution as it assumes that I have access to this organization. My regular user account is not part of this organization.

**Comment by jfullstackdev** [comment]
if you created the Org even in your second account, you can't simply leave without adding another user as the owner, that's the rule, so it's impossible that you don't know

if in any case, there is another owner, it's also impossible you don't know him too

you can try Support but there is no assurance

https://support.github.com/contact/account
